index.html

```diff
@@ -0,0 +1,856 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>New York Times Article Page</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+      rel="stylesheet"
+      integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <nav>
+      <ul>
+        <li><i class="fa fa-bars" aria-hidden="true"></i></li>
+        <li><i class="fa fa-search" aria-hidden="true"></i></li>
+        <li><a href="#">SPACE &amp; COSMOS</a></li>
+      </ul>
+      <img src="./assets/images/NewYork.PNG" alt="The New YorK Times" />
+      <div class="nav-right">
+        <a href="#">Subscribe now</a>
+        <button>Login</button>
+      </div>
+    </nav>
+
+    <main>
+      <section class="hero">
+        <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+        <img src="assets/images/18cosmos-superJumbo.jpg" alt="" />
+        <p>
+          Alan Guth was one of the first physicists to hypothesize the existence
+          of inflation, which explains how the universe expanded so uniformly
+          and so quickly in the instant after the Big Bang 13.8 billion years
+          ago.Credit... <span>Rick Friedman for The New York Times</span>
+        </p>
+      </section>
+
+      <section class="dennis">
+        <h4>By <a href="#!">Dennis Overbye</a></h4>
+
+        <div class="share">
+          <p>March 12, 1984</p>
+          <ul>
+            <li>
+              <a href=""><i class="fa fa-facebook social"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa fa-twitter social"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa fa-envelope social"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa fa-share social"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa fa-bookmark"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa fa-comment"></i></a>
+            </li>
+          </ul>
+        </div>
+        <div class="container">
+          <div class="main-paragraph">
```

Comment:
- The `main-paragraph` must be centered.
  
  Original 👇  
  ![Annotation 2020-06-09 222838](https://user-images.githubusercontent.com/57726348/84177682-d0a22d00-aaa0-11ea-9932-01f20eee7ebc.jpg)

  Right now 👇  
  ![Annotation 2020-06-09 222810](https://user-images.githubusercontent.com/57726348/84177712-e152a300-aaa0-11ea-9ba7-33e7ae7e9f71.jpg)